gcloud/logging/config_service_v2_api.py

```diff
@@ -14,64 +14,123 @@
 #
 # EDITING INSTRUCTIONS
 # This file was generated from the file
-# https://github.com/google/googleapis/blob/7710ead495227e80a0f06ceb66bdf3238d926f77/google/logging/v2/logging_config.proto,
+# https://github.com/google/googleapis/blob/master/google/logging/v2/logging_config.proto,
 # and updates to that file get reflected here through a refresh process.
 # For the short term, the refresh process will only be runnable by Google engineers.
-# Manual additions are allowed because the refresh process performs
-# a 3-way merge in order to preserve those manual additions. In order to not
-# break the refresh process, only certain types of modifications are
-# allowed.
 #
-# Allowed modifications:
-# 1. New methods (these should be added to the end of the class)
-#
-# Happy editing!
+# The only allowed edits are to method and file documentation. A 3-way
+# merge preserves those additions if the generated source changes.
+
+import os
+import platform
 
 from google.gax import api_callable
```

Comment:
weird question --- does the protobuf package install cleanly with this one? ive had problems in the past with the google namespace after running pip install protobuf...